client/app/components/packages/pas-form/edit.js

```diff
@@ -14,6 +17,13 @@ export default class PasFormComponent extends Component {
     packageInstance.saveDescendants();
   }
 
+  @action
+  async submit(packageInstance) {
```

Comment:
We've been using `projectPackage`. Couldn't find any existing instances of `packageInstance` as a parameter name

Reply:
okay will change

Reply:
okay, i lied, just one instance in the `edit.js` file.

but there's 47 uses of projectPackage.

this is a super nitpick :p

Reply:
nope you're right, i didn't like the naming there either. we know it's an instance anyways!